sonar-cxx-plugin/src/test/java/org/sonar/plugins/cxx/TestUtils.java

```diff
@@ -171,16 +172,16 @@ public static void addInputFile(DefaultFileSystem fs,
         temp.setCharAt(0, Character.toUpperCase(temp.charAt(0)));
         path = temp.toString();
       }
-      inputFile = new DefaultInputFile(path);
-      inputFile.setAbsolutePath(path);
+      inputFile = new DefaultInputFile("key", path);
```

Comment:
@Bertk @guwirth this seems to be a problem for us? now they require DefaultInputFile to behave as a module project. this is going to brings many issues to us